praw/models/reddit/mixins/inboxable.py

```diff
@@ -7,13 +7,7 @@ class InboxableMixin(object):
     """Interface for RedditBase classes that originate from the inbox."""
 
     def block(self):
```

Comment:
I wonder if we should deprecate this method now given that it makes more sense to do it directly on a Redditor. Let's leave it as is for now though -- food for thought for a future pull request.

Reply:
I didn't touch it because it is an API endpoint that exists separately, so it might have its uses. Specifically, it might save a request to use `'api/block'` instead of `'api/block_user'` if you have the fullname of an item in your inbox but don't have the fullname of its author.